core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -609,13 +609,13 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
     require(!encryptionEnabled || get(NETWORK_AUTH_ENABLED),
       s"${NETWORK_AUTH_ENABLED.key} must be enabled when enabling encryption.")
 
-    val executorTimeoutThreshold = getTimeAsSeconds("spark.network.timeout", "120s")
-    val executorHeartbeatInterval = getTimeAsSeconds("spark.executor.heartbeatInterval", "10s")
+    val executorTimeoutThreshold = getTimeAsMs("spark.network.timeout", "120s")
```

Comment:
Could you change `getTimeAsMs` back to `getTimeAsSeconds`? There is a slight difference when the user doesn't specify the time unit. `getTimeAsMs` uses `ms` as default, while `getTimeAsSeconds` uses `seconds`.

Reply:
This part in the code only does some validation logic on the values and possibly throws an error. Because we changed it to use get(EXECUTOR_HEARTBEAT_INTERVAL), which returns the value in ms, I wanted the other one to return value in ms as well so we're comparing values in the same unit.

Reply:
Could you use `getTimeAsSeconds` and manually convert it to `ms`?